media/css/mozorg/home/home-promo.less

```diff
@@ -1105,6 +1107,27 @@ html[lang|="en"] {
     }
 }
 
+html[lang="de"],
+html[lang="it"] {
```

Comment:
It looked like the initial state had a really long second line so, thought it best to lessen the size. Perhaps not needed.